### ISSUE TITLE
Widgets werden abgeschnitten bei unter 320 Pixel Breite

### ISSUE THREAD

**Issue by Zodiac1978** [Zodiac1978/meetup#1]
Bei einer Breite von 320 Pixeln werden die Widgets angeschnitten. Die Paddings gelten auch für die einspaltigen Elemente (wie #comments), daher wahrscheinlich die zusätzliche Breite durch die Zweispaltigkeit (und damit doppelten paddings) in der Mobilansicht.
Sichtbar im Firefox/Chrome/Safari bei unter 320 Pixel Breite und auf dem iPhone 4S (mein einziges mobiles Testobjekt - siehe Screenshot)
Du könntest die Widgets bei unter 320 Pixel einspaltig setzen (meine Präferenz) oder Du setzt die Breite der Widgets auf 85% statt 100%. Dann passt es bei mir.
# secondary .widget, #tertiary .widget {

```
 width: 85%;
```

}

![img_0316](https://f.cloud.github.com/assets/870475/743657/8ff1473c-e3ec-11e2-9297-a4f0dfc4b143.png)

**Comment by glueckpress** [comment]
Danke für den Vorschlag, werden einspaltig.

**Comment by Zodiac1978** [comment]
Gerne! :)

**Comment by glueckpress** [comment]
Nee, nee, schließen tue ich das schon selber, wenn's umgesetzt ist. ;)
